modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -1817,6 +1817,136 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12167. On the Product Details page user can see option values with two different SKUs linked"
+	@priority = 5
+	test CanViewOptionsValueWithTwoSKUsLinked {
+		property portal.acceptance = "true";
+
+		task ("Given a Minium site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And a simple product is created") {
+			CommerceJSONProductsAPI._addCommerceProduct(
+				catalogName = "Minium",
+				productName = "Test Simple Product",
+				productType = "Simple",
+				sku = "SIM1001");
+		}
+
+		task ("And a new option is created") {
```

Comment:
"And 2 SKU contributor static options with 2 values with a delta price, each linked to a SKU are created".

But I think this task is too big. It would be better to split it up into multiple tasks.

Reply:
Is the same task is only do it twice, it got a bit bigger because of the for, but they are doing the same thing